### ISSUE TITLE
Reassign one of the sensors from 4&5 to other analog pins so we can use both.

### ISSUE THREAD

**Issue by gianlazz** [sameezch/OMNI-BAND#4]
It seems to be assigned by default in the adafruit sensor library to analog pin 4&5 so it's not plainly stated in the sketch.

**Comment by gianlazz** [comment]
Is this the only I2C pins?

Connect to the Arduino:
The LSM303 breakout requires only 4 wires:
GND->GND
VIN->5v (It will also work on 3.3v)
SDA->SDA (A4 on 'classic' Arduinos)
SCL->SCL (A5 on 'classic' Arduinos)

**Comment by gianlazz** [comment]
These displays use I2C to communicate, 2 pins are required to interface

**Comment by gianlazz** [comment]
Topic: Where are the Arduino I2C ports? (Read 5963 times)
https://forum.arduino.cc/index.php?topic=108904.0

"The display uses I2C and I also believe the RTC uses I2C.  Now, my real question is, where am I supposed to hook the wires up to for both of these at the same time?  I see references to the I2C pins being A4 and A5 on the Arduino, but I was under the impression the I2C pins were on the top left, next to the AREF pin.  What is the difference between the A4-A5 I2C and the I2C pins next to the AREF?  My problem may be that I bussed the RTC into the top two pins instead of A4 and A5 and the code I am using is for the analog pins."

 **"A4/A5 are the I2C pins. Connect to all devices in parallel. Be sure to use ~4.7K pullup resistors on both lines."**

"There are no real "other" I2C pins. There are only one set. The ones marked SLC and SDA above the Aref pin are duplicates of what ever real pins hold the I2C signals on that board's processor. This is so shields can work with all sorts of boards.
If you have no pull up resistors on your arduino the you need to fit some externally. The internal pull ups are not sufficient especially with two devices on the bus."

@sameezch so the whole adafruit sensor library using only pins A4 & A5 seems to be due to those being the I2C pins for this microcontroller, which is the protocol both the temp sensor and accelerometer use to communicate. I'm confused because apparently so are the SCL & SDA pins on the other side of the board past digital pin 13. If I'm understanding correctly though, SCL & SDA are,/ or can be, connected to A4 & A5. So if you look at the bold quote above it looks like you can connect multiple I2C devices at once to the arduino's I2C pins by connecting them in parallel, though we also most likely will need to add another 4.7k pullup resistor so that it still registers both of the sensors since the onboard I2C pullup resistor most likely won't be strong enough for two devices.

**Comment by gianlazz** [comment]
@sameezch I also think once I fix this the main loop in the sketch will be able to finish and continue to print out the serial info like we were trying to solve. I think it may stop when it doesn't see any info for the accelerometer.

**Comment by gianlazz** [comment]
Yup hooking them up in parallel solved this issue and the main loop not completing.
